website/docs/core-concepts/connecting-to-data-sources/authentication/connect-to-apis.md

```diff
@@ -4,24 +4,102 @@ description: You can hit any REST endpoint available on the public internet
 
 # Configuring APIs
 
+Configuring APIs involves setting up and managing the various aspects of an API that allow it to function effectively. This can include defining the API's structure and capability, setting up authentication and authorization, and establishing connections with other systems or services.
+
 :::tip
-If your API is not available on the public internet, you must expose it via a service such as [https://ngrok.com/](https://ngrok.com/)
+If you want to access an API that's not publicly available on the internet, you can use a service like [ngrok](https://ngrok.com/) to expose it. For more information, see [How to work with local APIs on Appsmith](/advanced-concepts/more/how-to-work-with-local-apis-on-appsmith).
 :::
 
-## API Editor
+## API query editor
 
-The API pane is a REST interface that allows you to create and modify your existing APIs. All [REST HTTP methods](https://www.w3schools.in/restful-web-services/rest-methods) are supported and API values can be configured in the headers, params, and body fields
+The API pane is a REST interface that allows you to create and modify your existing APIs. All [REST HTTP methods](https://www.w3schools.in/restful-web-services/rest-methods) are supported and API values can be configured in the headers, Params, and body fields
 
 ![](</img/create_api_(1).gif>)
 
-## Importing CURL Commands
+## Configure API query
+
+Configuring an API query involves specifying the various components that make up the request and the desired response. Here is an overview of the main elements you need to consider when configuring an API query.
```

Comment:
Can we replace API with REST API ?

Reply:
done

Reply:
@sumitsum Hey do we need to mention REST everywhere? In the beginning and in the intro wouldn't it suffice to add REST and then avoid it in sub-headings?